stglib/core/utils.py

```diff
@@ -159,7 +159,7 @@ def add_min_max(ds):
     [exclude.append(k) for k in ds.variables if "time" in k]
     exclude.extend(["TIM", "TransMatrix"])
 
-    alloweddims = ["time", "sample", "depth"]
+    alloweddims = ["time", "sample", "depth","z"]
```

Comment:
Please make sure to run the "black" code formatting tool on your files. That will standardize the code style. There are a bunch of examples in this file where that needs to be changed.

Reply:
Done.